package.json

```diff
@@ -1,11 +1,10 @@
 {
   "name": "habitica-markdown",
-  "version": "1.3.2",
+  "version": "1.4.0",
   "description": "A markdown-it module pre-configured for use in Habitica",
   "main": "index.js",
   "scripts": {
-    "build": "npm install --force && node tasks/build.js && npm run minify",
-    "minify": "uglifyjs dist/habitica-markdown.js > dist/habitica-markdown.min.js",
+    "build": "npm install --force && webpack --config webpack.config.js --mode=${TARGET_ENV:-development}",
```

Comment:
I think build is only going to be used when publishing new versions so I would set the mode to `production` by default

Reply:
Also could you commit the updated dist files in this PR?

Reply:
Doesn't it make sense to have a development build for local linking? Or does that use the source code directly?

Also isn't the dist dir in git only necessary if you use bower?

Reply:
Yeah, the separate target seems to work well. I'd prefer to actually add the `dist` directory to the .gitignore or is it required to have those generated artifacts in the dist directory? If that's the case does it need the minified or the development version?

Reply:
Yeah the dev build looks good, I didn't think about it because modern apps can simply use `require('habitica-markdown`)` or `import 'habitica-markdown'`

> I'd prefer to actually add the dist directory to the .gitignore or is it required to have those generated artifacts in the dist directory? If that's the case does it need the minified or the development version?

Agreed but I'd like to make it as easy as downloading a file if someone wants to use it outside of a environment where modules are supported and if they don't know how to use webpack, it'll save on support time

Reply:
> Also isn't the dist dir in git only necessary if you use bower?
Not really, being built as an UMD library you can simply load the JS script in an HTML page and access the library at `window.habiticaMarkdonw`

> Doesn't it make sense to have a development build for local linking? Or does that use the source code directly?

The main habitica repo will use the source file directly (package.json -> main) by consuming the NPM package, for those that use the compiled version I'd say the minified version is fine since it comes with a source map

Reply:
Cool, re-added the dist directory containing the production files.